src/styles/_heading.scss

```
@@ -0,0 +1,40 @@
+.heading {
+  font-size: 48px;
+  line-height: 1;
+  position: absolute;
+  text-transform: uppercase;
+  font-weight: 300;
+  width: max-content ;
```

Упорядкуй це, для кращого розуміння, що тут знаходиться для позиціонування, а що для стилів 😃 
```suggestion
  position: absolute;
  width: max-content ;
  
  font-size: 48px;
  line-height: 1;
  text-transform: uppercase;
  font-weight: 300;
```